### ISSUE TITLE
Could not find a declaration file for module 'pino'

### ISSUE THREAD

**Issue by smelnikov** [WalletConnect/walletconnect-monorepo#1612]
**Describe the bug**
A typings error after installing the WalletConnect types file into my app.

**SDK Version (if relevant)**
"typescript": "^4.7.3",
"@walletconnect/sign-client": "^2.1.1",
"@walletconnect/types": "^2.1.1"

**To Reproduce**
To reproduce the behavior follow the instructions on https://docs.walletconnect.com/2.0/javascript/sign/installation

1. Create a new application
2. Install the above packages
3. Initialize the launch code from the installation page
4. Run script type check

Many errors of the same type appear:
```sh
node_modules/@walletconnect/types/dist/types/core/core.d.ts:1:24 - error TS7016: Could not find a declaration file for module 'pino'. '.../node_modules/pino/pino.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/pino` if it exists or add a new declaration (.d.ts) file containing `declare module 'pino';`

import { Logger } from "pino";
```

**Expected behavior**
There are no typing errors.

**Comment by smelnikov** [comment]
Is there some reason why the `pino@6.7.0` version is used in the `core` and `sign-client`?
Since `pino@7` they have included type declarations in the bundle and there are no type errors.

**Comment by smelnikov** [comment]
Also `pino@6` is no longer supported
https://github.com/pinojs/pino/blob/HEAD/docs/lts.md#schedule

**Comment by clemente-xyz** [comment]
I'm using `@walletconnect/sign-client` on React Native, and I'm also having an error when calling `SignClient.init` because `pino` is not defined:

```
TypeError: V.pino is not a function
```

Inspecting in my `node_modules`, looks like `@walletconnect/sign-client` does not install it properly.

Also, trying to pass a `logger` to avoid calling `pino` doesn't work.

Any fixes?

**Comment by bkrem** [comment]
Hi @ClementeSerrano,

This was just patched in 2.1.3 (see #1634 for context). Please let us know if you’re still experiencing this with the latest version.

The packages now also use `pino@7` throughout, so the original issue here should now hopefully be resolved @smelnikov

**Comment by clemente-xyz** [comment]
Hi @bkrem . Thank you very much. I was able to install the version.

Now, I'm having an unhandled exception, which is also generated on `SignClient.init()`, when somewhere in the WS process, `WsConnection` tries to access to `message`.

I attach a screenshot of the error and the trace that i can see on the console:

```
ExceptionsManager.js:180 TypeError: Cannot read properties of undefined (reading 'message')
    at parseConnectionError (error.js:50:14)
    at WsConnection.parseError (ws.js:125:36)
    at socket.onerror (ws.js:89:36)
    at WebSocket.dispatchEvent (event-target-shim.js:818:35)
    at WebSocket.js:271:14
    at EventEmitter.emit (_EventEmitter.js:149:33)
    at MessageQueue.__callFunction (MessageQueue.js:414:27)
    at MessageQueue.js:113:12
    at MessageQueue.__guard (MessageQueue.js:365:9)
    at MessageQueue.callFunctionReturnFlushedQueue (MessageQueue.js:112:10)
```

![Screenshot 2022-11-14 at 13 57 33](https://user-images.githubusercontent.com/9727036/201666767-487bfe0d-04fb-4e82-9695-007db7a35248.png)

I followed step by step the [React Native installation](https://docs.walletconnect.com/2.0/javascript/guides/react-native).

My project versions are:

```json
{
  //...
  "dependencies" : {
    // ...
    "react-native": "0.65.1",
    "@walletconnect/react-native-compat": "^2.1.3",
    "@walletconnect/sign-client": "^2.1.3",
    "@walletconnect/utils": "^2.1.3",
    "@react-native-async-storage/async-storage": "^1.17.11",
    "react-native-get-random-values": "^1.8.0",
  }
}
```

Any idea what is happening?

**Comment by bkrem** [comment]
> Now, I'm having an unhandled exception, which is also generated on SignClient.init(), when somewhere in the WS process, WsConnection tries to access to message.

@ClementeSerrano off the top of my head I'd make sure that a valid `projectId` value is being passed to `SignClient.init()` as shown in the docs.

This is unrelated to this issue though so please open a separate issue if you're unable to resolve.

**Comment by clemente-xyz** [comment]
ok, thanks!

**Comment by clemente-xyz** [comment]
@bkrem seems that https://github.com/WalletConnect/walletconnect-monorepo/issues/1519 is the same

should we continue there?

**Comment by smelnikov** [comment]
> Hi @ClementeSerrano,
> 
> This was just patched in 2.1.3 (see #1634 for context). Please let us know if you’re still experiencing this with the latest version.
> 
> The packages now also use `pino@7` throughout, so the original issue here should now hopefully be resolved @smelnikov

Yup, 2.1.3 looks legitimate
